src/Tribe/Attendees.php

```diff
@@ -820,6 +825,47 @@ public function maybe_generate_csv() {
 		}
 	}
 
+	/**
+	 * Admin requested a CSV export from the attendees table.
+	 * Generates the download and finishes the execution.
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function export_csv_for_all() {
+		if ( empty( $_GET['attendees_csv'] ) || empty( $_GET['attendees_csv_nonce'] ) || ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		// Generate filtered list of attendees.
+		$items = $this->generate_filtered_list( 'all' );
+
+		// Sanitize items for CSV usage.
+		$items = $this->sanitize_csv_rows( $items );
+
+		if ( ! empty( $items ) ) {
```

Comment:
It would be more readable to invert this conditional and then return early, instead of keeping all the subsequent logic within this if statement.